csharp/README.md

```diff
@@ -1,5 +1,16 @@
 # XenServer.NET usage examples
 
+## Overview
+
+The XenServer.NET examples are shipped as a Microsoft Visual Studio solution.
+It includes the following files:
+
+-  `GetVariousRecords`: logs to a XenServer host and displays information about 
```

Comment:
nit: logs in*
 
 VERY nit: start the bullet points with capital letters like in other READMEs? (same in the python readme)


same below